src/Components/Header/Header.js

```diff
@@ -1,12 +1,64 @@
 import React, { Component } from "react";
+import "./Header.scss";
 
 class Header extends Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
   render() {
-    return <></>;
+    return (
+      <>
+        <div className="header">
+          {/* user__Menu */}
+          <ul className="header__menu">
+            <li className="header__signup">
+              <a>회원가입</a>
+            </li>
+            <li className="header__login">
+              <a>로그인</a>
+            </li>
+            <li className="header__customer">
+              <a> 고객센터 </a>
+            </li>
+            <li className="header__search">
+              <a>배송지역 검색</a>
+            </li>
+          </ul>
+          {/* header__Logo */}
+          <div className="header__logo">
+            <img src="https://res.kurly.com/pc/service/common/1908/delivery_190819.gif"></img>
```

Comment:
- img 태그에 들어가는 alt 속성 또한 아래와 같은 이유로 매우 중요합니다. 
> 1. 이미지가 로딩되지 않을 경우 alt 속성에 적힌 값이 보여짐.
> 2. 시각장애인의 경우 alt 속성을 통해 어떤 이미지가 보여지는지 알 수 있음. 
> 3. SEO 검색엔진 최적화에서 어떤 img인지 인식하는 코드는 alt 속성임. 

- 위와 같은 이유로 alt속성을 중요하기 떄문에 src속성보다 먼저 작성하는게 좋습니다.
- jsx에서 content가 없는 요소는 self closing합니다.
```suggestion
            <img alt="kurly logo" src="https://res.kurly.com/pc/service/common/1908/delivery_190819.gif" />
```